server_fn/src/lib.rs

```diff
@@ -0,0 +1,450 @@
+#![forbid(unsafe_code)]
+#![deny(missing_docs)]
+
+//! # Leptos Server Functions
```

Comment:
This title can be more framework-agnostic if you'd like. Doesn't matter to me either way.